vhdbuilder/packer/generate-windows-vhd-configuration.ps1

```diff
@@ -41,7 +41,7 @@ if ($windowsSku -eq "2019") {
     $global:imagesToPull = @(
             "mcr.microsoft.com/windows/servercore:ltsc2019",
             "mcr.microsoft.com/windows/nanoserver:1809",
-            "mcr.microsoft.com/oss/kubernetes/pause:3.6-hotfix.20220114",
```

Comment:
Please do not remove the old one. We will switch pause image from 3.6-hotfix.20220114 to 3.9 in rp after the new vhd with this change is available in all regions. Before that, users should still use 3.6-hotfix.20220114